libevmasm/Assembly.cpp

```diff
@@ -598,6 +642,23 @@ LinkerObject const& Assembly::assemble() const
 			ret.linkReferences[ret.bytecode.size()] = m_libraries.at(i.data());
 			ret.bytecode.resize(ret.bytecode.size() + 20);
 			break;
+		case PushImmutable:
+			ret.bytecode.push_back(uint8_t(Instruction::PUSH32));
+			ret.immutableOccurrences[i.data()].emplace_back(ret.bytecode.size());
```

Comment:
Two different members of the same name are used for 1) number of occurences and 2) bytecode offsets. I'll try to disambiguate that.